packages/@aws-cdk/cx-api/lib/features.ts

```diff
@@ -21,14 +21,20 @@
 export const ENABLE_STACK_NAME_DUPLICATES_CONTEXT = '@aws-cdk/core:enableStackNameDuplicates';
 
 /**
- * IF this is set, `cdk diff` will always exit with 0.
+ * If this is set, `cdk diff` will always exit with 0.
  *
  * Use `cdk diff --fail` to exit with 1 if there's a diff.
  */
 export const ENABLE_DIFF_NO_FAIL_CONTEXT = 'aws-cdk:enableDiffNoFail';
 /** @deprecated use `ENABLE_DIFF_NO_FAIL_CONTEXT` */
 export const ENABLE_DIFF_NO_FAIL = ENABLE_DIFF_NO_FAIL_CONTEXT;
 
+/**
+ * If this is set, any newly created ECS service will have the logical name end
+ * with `Resource` rather than `Service`.
+ */
+export const ENABLE_CFN_SERVICE_RESOURCE_RENAME = 'aws-cdk/aws-ecs:enableCfnServiceResourceRename';
```

Comment:
Would prefer to call this something snappier and more to the point, like `shortLogicalId`.

Reply:
Add `_CONTEXT` to the variable name to mirror the `ENABLE_DIFF` example.

Reply:
Renamed it to ENABLE_SHORTEN_SRVC_LOGICAL_ID_CONTEXT any objection?